.github/workflows/test.yaml

```diff
@@ -76,16 +76,18 @@ jobs:
   kubernetes-tests:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: AbsaOSS/k3d-action@v1.5.0
+      - uses: actions/checkout@v3
+
+      - uses: AbsaOSS/k3d-action@v2
         name: "Create k3d Cluster"
         with:
           cluster-name: "k3s-default"
           args: >-
-            -p "30200:30200@agent[0]"
+            --port "30200:30200@agent:0:direct"
```

Comment:
This related to breaking changes in bumping the k3d action, which now used k3d v5 instead of k3d v4.